dist/index.js

```diff
@@ -1317,11 +1318,12 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const fs = __importStar(__webpack_require__(747));
 const dotenv_safe_1 = __importDefault(__webpack_require__(245));
 const core = __importStar(__webpack_require__(470));
-const parseEnv = (envPath, examplePath) => {
+const parseEnv = (envPath, examplePath, allowEmpty = "false") => {
```

Comment:
Having a stringified boolean here is a little weird, what do you think of changing this to a boolean and do the boolean conversion before calling the function.